planetiler-core/src/main/java/com/onthegomap/planetiler/writer/TileArchive.java

```diff
@@ -24,6 +25,11 @@ default void printStats() {}
   }
 
 
+  /**
+   * specify the preferred insertion order for this archive, e.g. TMS or HILBERT. See {@link TileOrder}.
```

Comment:
One neat thing you can do in javadoc too is link directly to class members:

```suggestion
   * Specify the preferred insertion order for this archive, e.g. {@link TileOrder#TMS} or {@link TileOrder#HILBERT}.
```